Makefile

```diff
@@ -1,6 +1,16 @@
 REGISTRY ?= quay.io/kubevirt
 IMAGE_TAG ?= latest
 IMAGE_GIT_TAG ?= $(shell git describe --abbrev=8 --tags)
+PLATFORMS ?= linux/amd64,linux/s390x
+# Set the platforms for building a multi-platform supported image.
+# Example:
+# PLATFORMS ?= linux/amd64,linux/arm64,linux/s390x
+# Alternatively, you can export the PLATFORMS variable like this:
+# export PLATFORMS=linux/arm64,linux/s390x,linux/amd64
+ARCH := $(shell uname -m | sed 's/x86_64/amd64/')
+DOCKER_BUILDER ?= marker-docker-builder
+MARKER_IMAGE_TAGGED_1 := ${REGISTRY}/bridge-marker:${IMAGE_TAG}
+MARKER_IMAGE_TAGGED_2 := ${REGISTRY}/bridge-marker:${IMAGE_GIT_TAG}
```

Comment:
please renamed to `MARKER_IMAGE_TAGGED`
and `MARKER_IMAGE_GIT_TAGGED`

Reply:
Done.